chaos-monkey-spring-boot/pom.xml

```diff
@@ -126,6 +126,14 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>no.finn.unleash</groupId>
+            <artifactId>unleash-client-java</artifactId>
+            <version>4.2.1</version>
+            <optional>true</optional>
```

Comment:
Note this is an `optional` dep. So users of the library won't get Unleash automatically. This similar approach is used by the Micrometer project.